database/executor/migration_up/main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/mysql"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	"log"
+	"myapp/database"
+)
+
+func main() {
```

Comment:
jangan ada 3 main, bs pake CLI aja jd 1 aja main file nya 

https://dev.to/divrhino/adding-flags-to-a-command-line-tool-built-with-go-and-cobra-34f1
https://github.com/spf13/cobra/tree/main

Reply:
bisa liat source krakenD jg di `executeo.go`

Reply:
done, please check new code